From b961b20d042541a558ec76446be02113783fc818 Mon Sep 17 00:00:00 2001
From: Grzegorz Jaszczyk <jaz@semihalf.com>
Date: Thu, 21 Feb 2019 11:19:54 +0100
Subject: [PATCH 1029/1051] phy: core: remove rxaui mode from generic phy

Since the RXAUI mode was introduced as a submode of Etheret phy mode,
remove it from generic phy modes list.

Change-Id: I22a55abc17413f80d08a61ec7ae023c1bd9c5528
Signed-off-by: Grzegorz Jaszczyk <jaz@semihalf.com>
Reviewed-on: https://sj1git1.cavium.com/4645
Reviewed-by: Stefan Chulski <Stefan.Chulski@cavium.com>
Reviewed-by: Marcin Wojtas <Marcin.Wojtas@cavium.com>
Tested-by: sa_ip-sw-jenkins
[Kevin: The original patch got from Marvell sdk10.0_19.06]
Signed-off-by: Kevin Hao <kexin.hao@windriver.com>
---
 include/linux/phy/phy.h | 1 -
 1 file changed, 1 deletion(-)

diff --git a/include/linux/phy/phy.h b/include/linux/phy/phy.h
index d2e94a939e7d..2bb544e8863e 100644
--- a/include/linux/phy/phy.h
+++ b/include/linux/phy/phy.h
@@ -39,7 +39,6 @@ enum phy_mode {
 	PHY_MODE_UFS_HS_B,
 	PHY_MODE_PCIE,
 	PHY_MODE_SATA,
-	PHY_MODE_RXAUI,
 	PHY_MODE_ETHERNET,
 };
 
-- 
2.17.1

